### ISSUE TITLE
Simulated-only run

### ISSUE THREAD

**Issue by espetro** [carter-james89/UnityControllerForTello#3]
Hey there,

Thanks for making such an awesome package for Unity and the Tello. So far, from what I’ve seen in the videos, you can run the autopilot by connecting to the real drone first. Is there a way to play/test with the drone just using the simulated device, so if it crashes in Unity, it doesn’t in real life?

Thanks in advance!

**Comment by carter-james89** [comment]
yes, you just need to set the control to simulated mode and it will work without being connected to the tello
